Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:10.15.3-alpine
 
 # Create app directory
-WORKDIR /mine-support
+WORKDIR /code
```

Comment:
It was for consistency across the projects, rather than having a folder name unique to the project.
The benfit for me is when mounting local folders in docker-compose it's more obvious what is being mounted.

We don't have in depth guidelines on docker files, but there a few notes in https://eaflood.atlassian.net/wiki/spaces/FFCPD/pages/1297219944/Standards+for+Development+Project+Structure+and+Naming+Conventions

If the team is happy with a move to having 'code' as the folder name, we could update the above page with the view to adding it to the docker standards of the developer guide https://defradev.visualstudio.com/DEFRA_FutureFarming/_git/defra-ff-documentation?path=%2Fdeveloper-guide%2Findex.md&version=GBmaster&createIfNew=true&_a=preview in future

Reply:
I'm happy with code.  The less things we have to think about the better and agree it should go in the Developer guide.  There is a planned section for Docker, we can include it when we complete that ticket.

Reply:
The convention seems to be /usr/src/app and I'd expect we'd follow that

I think we've got to also remember we're talking about creating folders on a Linux filesystem, so putting a folder under root is probably not something we should be doing given conventions in that space.

That said, /usr/src/app breaks Linux conventions, but it's broadly accepted (see https://github.com/nodejs/docker-node/issues/96)

TL;DR /code makes my eyes bleed!

Reply:
Linked issue is interesting, seems some tools are dependent on us using that folder, could be something we need to use in the project is similar so we could avoid headaches later by following the convention.

As long as we're consistent and it's documented we still get the benefits @markysoft  has stated so I'm happy to go with the convention of /usr/src/app.

Reply:
Seems to be a few thoughts on this, all different to our initial ones!
The official node.js sample uses `/usr/src/app`, with the caveats above, while others prefer `/opt`

I'm happy with the suggestions and the reasoning, and will change the WORKDIR to `/usr/src/app`